source/dub/packagemanager.d

```diff
@@ -305,8 +318,12 @@ class PackageManager {
 		}
 
 		string gitReference = versionSpec.chompPrefix("~");
-		const destination = m_repositories[LocalPackageType.user].packagePath ~
-			NativePath(name ~ "-" ~ gitReference) ~ (name~"/");
+		NativePath destination = getPackagePath(
+			m_repositories[LocalPackageType.user].packagePath,
+			name, versionSpec);
```

Comment:
destination changed here, is that right? (using versionSpec directly instead of gitReference here)

Reply:
`getPackagePath` also `chompPrefix`, so that didn't change.
However `getPackagePath` also replaces `+` with `_`, but git commit hash shouldn't have any `+`.